mdbenchmark/tests/test_utils.py

```diff
@@ -145,3 +151,62 @@ def dummy(arg):
     utils.guess_ncores()
     out, err = capsys.readouterr()
     assert out == output
+
+
+def test_DataFrameFromBundle(data):
+    """Test DataFrameFromBundle function.
+       This is used in other tests, therefore everything is hard coded
+       If changes are made to the layout type this should be changed here.
+    """
+    bundle = mds.discover(data["analyze-files-gromacs"])
+    test_output = utils.DataFrameFromBundle(bundle)
+
+    expected_output = pd.read_csv(data["analyze-files-gromacs.csv"])
+
+    # TODO: This test fails if we test the dtype. This is weird and I would
```

Comment:
Is this TODO still relevant?

Reply:
ah yeah I couldn't figure out why this happens yet. But currently the tests are fine and this is no major problem